### ISSUE TITLE
Bypassable?

### ISSUE THREAD

**Issue by Ruichka** [L0615T1C5-216AC-9437/GlobalImageBan#1]
```java
if (lb.code.contains("drawflush display")) {
```

Wouldn't it be bypassable by `getlink`ing a display to another variable and then doing `drawflush totallyNotAScreen`?

**Comment by L0615T1C5-216AC-9437** [comment]
yes, but afaik there is no mod that does that right now. If it were to happen in the future and becomes wide spread, I would likely code a parser that detects such roundabout method of injecting images into displays.
Edit 1: Fix punctuation
